third-party/test/index.ts

```diff
@@ -71,6 +70,8 @@ export function defaultMochaOptions(useColors: boolean = true): MochaOptions {
             }
         }
     }
+    ;(o as any).color = useColors
```

Comment:
workaround the fact that `"@types/mocha": "^5.2.0"` in `package.json` is mismatched with `"mocha": "^6.2.0"`. Not sure how to fix that: `npm install` claims that `@types/mocha` versions 6.2.0 and 6.0.0 do not exist.